### PR TITLE
feat: add variable "server.readinessProbe.Scheme" to argo chart

### PR DESCRIPTION
**Pull request by rtang03**
Checklist:

* [ X ] I have update the chart version in `Chart.yaml` following Semantic Versioning.
* [ X ] Any new values are backwards compatible and/or have sensible default.
* [ X ] I have followed the testing instructions in the [contributing guide](https://github.com/argoproj/argo-helm/blob/master/CONTRIBUTING.md).
* [ X ] I have signed the CLA and the build is green.
* [ X ] I will test my changes again once merged to master and published.

Changes are automatically published when merged to `master`. They are not published on branches.

### CODE REVIEW

File: charts/argo/templates/server-deployment.yaml [hunk 1]
```diff
@@ -52,7 +52,7 @@ spec:
             httpGet:
               path: /
               port: 2746
-              scheme: HTTP
+              scheme: {{ .Values.server.readinessProbeScheme }}
```
Comment (stefansedich) [inline charts/argo/templates/server-deployment.yaml:55]: Could we change this to `readinessProbe.scheme`.

File: charts/argo/values.yaml [hunk 2]
```diff
@@ -157,6 +157,8 @@ server:
   # SecurityContext to set on the server pods
   podSecurityContext: {}
   name: server
+  # either HTTP or HTTPS
+  readinessProbeScheme: HTTP
```
Comment (stefansedich) [inline charts/argo/values.yaml:161]: As above can we change this `readinessProbe.scheme`.

**Review by stefansedich — CHANGES_REQUESTED** [review]
Can we change to use `readinessProbe.scheme` that appears more common with charts for controlling the readiness probe.

File: charts/argo/values.yaml (hunk 2 above)
Comment (rtang03) [reply charts/argo/values.yaml:161]: Sure.

File: charts/argo/templates/server-deployment.yaml (hunk 1 above)
Comment (rtang03) [reply charts/argo/templates/server-deployment.yaml:55]: Sure. Should I submit another PR, to make above change?

**Comment by josephmcgovern-wf** [comment]
The core issue here may have already been solved by #506

**Comment by rtang03** [comment]
> The core issue here may have already been solved by #506

That is exactly what I want. And, this PR no longer requires.

**Pull request closed without merging**